src/Apps/CollectorProfile/Routes/Saves/Components/ArtworkListContent.tsx

```diff
@@ -55,14 +56,14 @@ function isContentOutOfView() {
 }
 
 const ArtworkListContent: FC<ArtworkListContentProps> = ({ me, relay }) => {
+  const { t } = useTranslation()
   const { match } = useRouter()
 
   const { jumpTo } = useJump()
   const { artworkListItemHasBeenTouched } = useArtworkListVisibilityContext()
-
-  const artworkList = me.artworkList!
+  const artworkList = me.artworkList
```

Comment:
I do support this change as I also thought that this enforcement is strange here when I looked at it first time, but with this change expected to see some more `?.` added down the file. Should we update all the usages for this `artworkList`?

Reply:
no more lint errors, it would block the git push/commit.